src/components/Buttons/ButtonBase/index.tsx

```diff
@@ -46,9 +43,11 @@ const ButtonBase = (props: ButtonBaseProps) => {
 
     return (
         <MaterialButton
+            sx={{
+                borderRadius: "calc(1rem + 6px)",
+            }}
```

Comment:
Why do we need this?

Reply:
It is from the old code of @thienduc12311 to make the styling like in Reply design.